burgle.lic

```diff
@@ -123,7 +123,7 @@ class Burgle
       return false
     end
     if @burgle_settings['max_search_count'] > 0
-      if /(already open|^You.+open)/ !~ bput("open my #{@loot_container}", 'already open', '^You.+open', '^You spread your arms, carefully holding your bag well away from your body', 'Please rephrase that command', 'What were you referring')
+      if /(already open|^You.+open)|^You can't do that./ !~ bput("open my #{@loot_container}", 'already open', '^You.+open', '^You spread your arms, carefully holding your bag well away from your body', 'Please rephrase that command', 'What were you referring', /You can't do that./)
```

Comment:
💡 The new phrase "You can't do that" can be included in the existing `(already open|^You.+open)` parentheses, same boolean logic applies.

💡 Also, the match strings passed to `bput` will be converted to regex so I'm suggesting `/You can't do that./` be `'You can't do that'` to match the existing format of the other match strings.

💡 Also, I'm suggesting removing the `.` from the new match string since as regex an unescaped `.` means any character and has an non-impactful influence in this scenario.

```suggestion
      if /(already open|^You.+open|^You can't do that)/ !~ bput("open my #{@loot_container}", 'already open', '^You.+open', '^You spread your arms, carefully holding your bag well away from your body', 'Please rephrase that command', 'What were you referring', 'You can't do that')
```

Reply:
You need to either escape the apostrophe or use the slashes I did. 

I can move into the parentheses. I can remove the period as well.